UtopiaCity/libman.json

```diff
@@ -0,0 +1,10 @@
+{
+  "version": "1.0",
+  "defaultProvider": "cdnjs",
+  "libraries": [
+    {
+      "library": "jqueryui@1.12.1",
```

Comment:
I will probably ask you to wait with this merge until lecture so that we can discuss jquery itself

Reply:
okey i will wait and fix the marks